floodviz/static/js/hydrograph.js

```diff
@@ -105,79 +132,121 @@
 				.attr("class", "axis")
 				.attr("transform", "translate(0," + height + ")")
 				.call(d3.axisBottom(x).tickFormat(d3.timeFormat("%B %e")));
+
 			// Add the Y Axis
 			svg.append("g")
 				.attr("class", "axis")
 				.call(d3.axisLeft(y).ticks(10, ".0f"));
+
 			// Tooltip
 			focus = svg.append("g")
-					.attr("transform", "translate(-100,-100)")
-					.attr("class", "focus");
+				.attr("transform", "translate(-100,-100)")
+				.attr("class", "focus");
+
 			focus.append("circle")
-					.attr("r", 3.5);
+				.attr("r", 3.5);
+
 			focus.append("text")
-					.attr("y", -10);
+				.attr("y", -10);
+
 			// Voronoi Layer
 			voronoi_group = svg.append("g")
-					.attr("class", "voronoi");
+				.attr("class", "voronoi");
 			voronoi_group.selectAll("path")
-					.data(voronoi.polygons(d3.merge(dataNest.map(function (d) {
-						return d.values
-					}))))
-					.enter().append("path")
-					.attr("d", function (d) {
-						return d ? "M" + d.join("L") + "Z" : null;
-					})
-					.on("mouseover", self.mouseover)
-					.on("mouseout", self.mouseout)
-					.on("click", function(d) {return self.click(d.data.key, graph_data)})
+				.data(voronoi.polygons(d3.merge(dataNest.map(function (d) {
+					return d.values
+				}))))
+				.enter().append("path")
+				.attr("d", function (d) {
+					return d ? "M" + d.join("L") + "Z" : null;
+				})
+				.on("mouseover", self.mouseover)
+				.on("mouseout", self.mouseout)
+				.on("click", function (d) {return self.click(d)})
 		};
-    
+
 		/**
 		 * Initalize the Hydrograph
 		 */
-		self.init = function() {
+		self.init = function () {
 			var data_path = options.data_path;
 			// Get the data
 			d3.json(data_path, function (error, data) {
+				if (error) { console.error(error); }
```

Comment:
I didn't really mean to make the code fault-tolerant, I just wanted it to log an error if it happens.